app/lib/theme/widgets/SelectParentWidget.tsx

```diff
@@ -84,11 +84,19 @@ const SelectParentWidget: React.FunctionComponent<
           );
         })}
       </Dropdown>
-      <style jsx>
+      <style jsx global>
         {`
           :global(input) {
             width: 100%;
           }
+          .pg-select-wrapper{
+            padding: 9px 2px 9px 2px;
+            margin-bottom: 5px;
+            margin-top:2px;
+          }
+          .pg-select-input {
+            color:#8e8e8e;
+          }
         `}
       </style>
```

Comment:
We want to avoid `style jsx global` as it removes the scoping from these styles, meaning that they might have side-effects and affect other components on the page. Now, I don't think we've been using [`:global()`](https://github.com/vercel/styled-jsx#one-off-global-selectors) properly either: I think that we should have an outer `div` (instead of a fragment) and have the following styles:

```suggestion
      <style jsx>
        {`
          div :global(input) {
            width: 100%;
          }
          div :global(.pg-select-wrapper) {
            padding: 9px 2px 9px 2px;
            margin-bottom: 5px;
            margin-top:2px;
          }
          div :global(.pg-select-input) {
            color:#8e8e8e;
          }
        `}
      </style>
```

The `div` part of the selector will be scoped to this specific component, and `:global` allows to select classes that are rendered by children components